lib/internal/test_runner/harness.js

```diff
@@ -22,12 +21,13 @@ const {
   parseCommandLine,
   setupTestReporters,
 } = require('internal/test_runner/utils');
+const { setImmediate } = require('timers');
 const { bigint: hrtime } = process.hrtime;
 
 const testResources = new SafeMap();
 
 function createTestTree(options = kEmptyObject) {
-  return setup(new Test({ __proto__: null, ...options, name: '<root>' }));
+  return setup(new Suite({ __proto__: null, ...options, name: '<root>' }));
```

Comment:
I had in mind that we need to do it sometime

Does that fix the following issue:
- #48913

Reply:
IDK

Reply:
I've just tested. it does not